doc/Process_Console.md

```diff
@@ -0,0 +1,595 @@
+ Process Console
+ ===============
+ 
+ `process_console` is a capsule that implements a small shell over the UART that allows
+ a terminal to inspect the kernel and control userspace processes.
+
+  - [Setup](#setup)
+  - [Using Process Console](#using-processconsole)
+  - [Commands](#commands)
```

Comment:
Please wrap this in the ToC comments (see another doc/ markdown file for an example).